refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/JavaWhereUsedQueryPlugin.java

```diff
@@ -104,7 +104,7 @@ public Problem preCheck() {
 
     @Override
     protected Problem preCheck(CompilationController javac) throws IOException {
-        javac.toPhase(JavaSource.Phase.PARSED);
+        javac.toPhase(JavaSource.Phase.ELEMENTS_RESOLVED);
```

Comment:
This is the change that fixes the problem - the rest is only test and support for tests.